lang/docs/src/doc.rs

```diff
@@ -1,127 +1,48 @@
 use std::fs;
-use std::io;
 use std::io::prelude::*;
-use std::path::Path;
-use std::path::PathBuf;
+use std::path::{Path, PathBuf};
 
+use askama::Template;
 use opener;
 
-use driver::paths::CSS_PATH;
+use driver::paths::{CSS_PATH, CSS_TEMPLATE_PATH};
 use driver::Database;
 use printer::{Print, PrintCfg};
 
-const HTML_END: &str = " </code></pre>
-    </div></body></html>";
-
-fn html_start(filepath: &Path) -> String {
-    format!(
-        "<!DOCTYPE html>
-<html lang=\"en\">
-<head>
-    <meta charset=\"UTF-8\">
-    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0\">
-    <title>{filename}</title>
-    <link rel=\"stylesheet\" href=\"style.css\">
-</head>
-<body>
-<div>
-        <h1>{filename}</h1>
-        <pre><code>",
-        filename = filepath.file_name().unwrap().to_string_lossy()
-    )
-}
-
 pub async fn write_html(filepath: &PathBuf, htmlpath: &PathBuf) {
     let mut db = Database::from_path(filepath);
     let uri = db.resolve_path(filepath).expect("Failed to resolve path");
     let prg = db.ust(&uri).await.expect("Failed to get UST");
     let cfg = PrintCfg::default();
 
     if !Path::new(CSS_PATH).exists() {
-        write_css();
+        fs::create_dir_all(Path::new(CSS_PATH).parent().unwrap())
+            .expect("Failed to create CSS directory");
+        fs::write(CSS_PATH, CSS_TEMPLATE_PATH).expect("Failed to create CSS file");
     }
+
     let mut stream = Box::new(fs::File::create(htmlpath).expect("Failed to create file"));
```

Comment:
`Box`ing the stream is unnecessary here. The reason this is done in other commands is because they support other output options (e.g. `stdout`).

```suggestion
    let mut stream = fs::File::create(htmlpath).expect("Failed to create file");
```